cw/cw305/capture_traces.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import binascii
+from Crypto.Cipher import AES
+import numpy as np
+import time
+from tqdm import tqdm
+import yaml
+
+import chipwhisperer as cw
+
+from util import device
+from util import plot
+
+
+def initialize_test(device_cfg):
+  """Initialize test setup."""
+  ot = device.OpenTitan(
+    device_cfg['fpga_bitstream'],
+    device_cfg['fw_bin'],
+    device_cfg['pll_frequency'],
+    device_cfg['baudrate'])
+  ot.initialize()
+  print('Scope setup with sampling rate {} S/s'.format(ot.scope.clock.adc_rate))
```

Comment:
How about using `f''` instead of `''.format()`, i.e. `f'Scope setup with sampling rate {ot.scope.clock.adc_rate} S/s'`?

Reply:
Done. Updated all strings to use the suggested format approach.